### PR TITLE
Bump sdoc to allow for json version bump

### PR DESCRIPTION
**Pull request by nciemniak**
Security vulnerability exists in current `json` version; `sdoc` needs to be bumped to allow us to use updated json version.

This PR resolves https://github.com/zooniverse/talk-api/issues/243 .

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -15,7 +15,7 @@ gem 'aws-sdk', '~> 2.3.7'
 gem 'faraday', '~> 0.9.2'
 gem 'faraday_middleware', '~> 0.12.2'
 gem 'pundit', '~> 1.1.0'
-gem 'sdoc', '~> 0.4', group: :doc
+gem 'sdoc', '>= 1.1.0', group: :doc
```
Comment (camallen) [inline Gemfile:18]: I don't even think this repo uses sdoc, can we remove it?
https://github.com/zooniverse/talk-api/search?q=sdoc

**Review by camallen — COMMENTED** [review]
I'd be in favour of removing this gem completely if it's not being used.

File: Gemfile (hunk 1 above)
Comment (nciemniak) [reply Gemfile:18]: Valid. I'm not aware of it being used, but this is also the first time I've touched this repo... the repo search does indeed seem to indicate that it is not in use, I will go ahead and remove.

**Review by camallen — COMMENTED** [review]
I think you will need to run `bundle install` to remove the gem from the `Gemfile.lock` as well.

**Comment by nciemniak** [comment]
@camallen there we go. Gemfile.lock updated. Forgot about this one for a sec there but we should be good to go now.

**Review by camallen — APPROVED** [review]
LGTM :shipit:

**Pull request merged**